app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewSerializer < ActiveModel::Serializer
+  attributes :id, :rating, :comment, :user, :user_id, :votes
+
+  # belongs_to :trail
+  has_many :likes
+
+  def votes
+    object.votes_count
+  end
+
+  def user
+    User.find(object.user_id).username
+  end
+
+  def user_id
+    object.user_id
+  end
+
+  def likes
+    object.likes
```

Comment:
Is there a reason you decided not to just hand through the ONE like for the current user here?

Reply:
@Brianna  - I've tried to fix this many times, but current_user (or a variant thereof) doesn't appear to be accessible in the ReviewSerializer class.